docs/authentication/configuration/sign-up-sign-in-options.mdx

```diff
@@ -5,18 +5,9 @@ description: Clerk provides various options for setting up a sign-up and sign-in
 
 # Sign-up and sign-in options
 
-Clerk provides various options for setting up a sign-up and sign-in flow. When a new application is created, you will be presented with the following application configuration screen:
+Clerk provides various options for setting up a sign-up and sign-in flow for your application, such as [identifiers](#identifiers), [authentication strategies](#authentication-strategies), [verification methods](#verification-methods), [social providers](##social-connections-o-auth), [Web3 authentication](#web3-authentication), [multi-factor authentication](#multi-factor-authentication), and [sign-up restrictions](#sign-up-restrictions). This guide will walk you through each of these options.
```

Comment:
I have two issues with this change:

1. Why recreate the TOC? If we change any of the headings we'll have to update this list every time, and it'll look sloppy if we ever forget.
2. An intro should introduce the page as succinctly as possible. I don't think a comprehensive list is necessary to introduce what this page is about.